docs/hub/spaces-sdks.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Spaces SDKs
+---
+
+<h1>Spaces SDKs</h1>
+
+Hugging Face Spaces officially supports three different SDKs: Gradio, Streamlit, and static HTML. This section gives a brief overview of the three methods for creating a Space.
+
+Gradio and Streamlit make it easy to quickly build apps using Python, and we recommend you try both out because they're really awesome! For in-depth information about using them, you can refer to their respective documentation.
+
+## Gradio
+
+**Gradio** provides an easy and intuitive interface for running a model from a list of inputs and displaying the outputs. For more details, take a look at this [tutorial](https://huggingface.co/blog/gradio) from the Gradio team about building GUIs for Hugging Face models.
+
+Selecting **Gradio** as the SDK when [creating a new Space](https://huggingface.co/new-space) will initialize your Space with the latest version of Gradio by setting the `sdk` property to `gradio` in your `README.md` file's YAML block. If you'd like to change the Gradio version, you can edit the `sdk_version` property.
+
+Visit the [Gradio documentation](https://gradio.app/getting_started) to learn all about its features and check out the [Gradio Guides](https://gradio.app/guides/) for some handy tutorials to help you get started!
+
+### Managing long queue times
+
+Since it's possible that many users could use your Space concurrently, we recommend using [Gradio's parameter for queueing predictions](https://gradio.app/getting_started/#queuing-to-manage-long-inference-times).
+
+## Streamlit
+
+**Streamlit** gives users more freedom to build a full-featured web app with Python in a *reactive* way. Your code is rerun each time the state of the app changes. Streamlit is also great for data visualization and supports several charting libraries such as Bokeh, Plotly, and Altair. Read our [blog post](https://huggingface.co/blog/streamlit-spaces) about building and hosting Streamlit apps in Spaces.
+
+To use Streamlit in a Space, you can select **Streamlit** as the SDK when you create a Space through the [**New Space** form](https://huggingface.co/new-space). This will create a repository with a `README.md` that contains the following properties in the YAML configuration block:
```

Comment:
```suggestion
To use Streamlit in a Space, select **Streamlit** as the SDK when you create a Space through the [**New Space** form](https://huggingface.co/new-space). This will create a repository with a `README.md` that contains the following properties in the YAML configuration block:
```